examples/plugins/BUILD.bazel

```diff
@@ -0,0 +1,64 @@
+load("@aspect_bazel_lib//lib:write_source_files.bzl", "write_source_files")
+load("@aspect_rules_swc//swc:defs.bzl", "swc", "swc_plugin")
+
+swc_plugin(
+    name = "npm_plugin",
+    src = "//:node_modules/@swc/plugin-transform-imports",
+    config = {
+        "lodash": {
+            "transform": "lodash/{{member}}",
+        },
+    },
+)
+
+swc_plugin(
+    name = "file_plugin",
+    src = "//examples/plugins/plugin-transform-imports:swc_plugin_transform_imports.wasm",
+    config = json.encode({
```

Comment:
How would a normal swc user configure this thing? Wouldn't they have a JSON file already in the project, or put this config inside the `.swcrc` file? It seems like a burden to make them put JSON string content into a BUILD file.

Reply:
Normally it'd go in the `.swcrc`. Having the config outside of the `.swcrc` is unavoidable here because SWC does not support relative plugin paths in the config and upstream was not interested in taking a patch to add support.

 I have added support for relative plugin paths when passing a `--config` option to the SWC CLI in https://github.com/swc-project/swc/pull/6835 but it has not been merged yet, and so this PR is blocked.

I'm open to other suggestions, the critical issue here is that SWC assumes Node.js-style package resolution based on climbing to a `node_modules` directory by default. Absolute paths are also supported, but obviously Bazel is unfriendly to that and I was trying to avoid having a wrapper script parse/rewrite the config before exec'ing SWC. I'm open to other suggestions!

Reply:
I guess now that your relative config option has landed, this can change to the desired API? Neat

Reply:
This is the API I ended up going with, given that plugin config is injected through a CLI argument. Let me know if you'd prefer a different approach.

Reply:
Done.